<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>248</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>2048</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(251, 248, 239);</string>
  </property>
  <widget class="QPushButton" name="pBtnLogin">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>580</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: rgb(115, 110, 100);
	background-color: rgb(251, 248, 239);
	border-radius: 5px;
}

QPushButton:hover
{
	background-color: rgb(219, 210, 200);
}

QPushButton:pressed
{
	background-color: rgb(187, 173, 160);
	padding-left: 3px;
	padding-top: 3px;
}</string>
   </property>
   <property name="text">
    <string>登 录</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnLogon">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>515</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: rgb(115, 110, 100);
	background-color: rgb(251, 248, 239);
	border-radius: 5px;
}

QPushButton:hover
{
	background-color: rgb(219, 210, 200);
}

QPushButton:pressed
{
	background-color: rgb(187, 173, 160);
	padding-left: 3px;
	padding-top: 3px;
}</string>
   </property>
   <property name="text">
    <string>注册账号</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnForget">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>515</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color: rgb(115, 110, 100);
	background-color: rgb(251, 248, 239);
	border-radius: 5px;
}

QPushButton:hover
{
	background-color: rgb(219, 210, 200);
}

QPushButton:pressed
{
	background-color: rgb(187, 173, 160);
	padding-left: 3px;
	padding-top: 3px;
}</string>
   </property>
   <property name="text">
    <string>忘记密码</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>240</y>
     <width>400</width>
     <height>240</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(219, 210, 200);
border-radius: 5px; </string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="lblPassword">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>110</y>
      <width>70</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>隶书</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(115, 110, 100);</string>
    </property>
    <property name="text">
     <string>密码</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ldtPassword">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>110</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(115, 110, 100);
background-color: rgb(219, 210, 200);
border: 1px solid gray;
border-radius: 5px; </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>enter password</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblID">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>40</y>
      <width>70</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>隶书</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(115, 110, 100);</string>
    </property>
    <property name="text">
     <string>账号</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ldtID">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>40</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(115, 110, 100);
background-color: rgb(219, 210, 200);
border: 1px solid gray;
border-radius: 5px; </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>enter game account</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbxID">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>隶书</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox{
	color: rgb(115, 110, 100);
}
QCheckBox::indicator {
    width: 13px;/*勾选框的大小*/
    height: 13px;
}</string>
    </property>
    <property name="text">
     <string>记住账号</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbxPswd">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>隶书</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox{
	color: rgb(115, 110, 100);
}
QCheckBox::indicator {
    width: 13px;/*勾选框的大小*/
    height: 13px;
	
	//color: rgb(115, 110, 100);
	//background-color: rgb(187, 173, 160);
}</string>
    </property>
    <property name="text">
     <string>记住密码</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>115</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Algerian</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 110, 100);</string>
   </property>
   <property name="text">
    <string>2048</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblLogin">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>105</y>
     <width>240</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 110, 100);</string>
   </property>
   <property name="text">
    <string>登 录</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblIcon">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/icon.jpeg);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtnRet">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>120</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>返回上一级</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border-image: url(:/interface/images/home.png);
	background-color: rgb(251, 248, 239);
	border-radius: 5px;
}

QPushButton:hover
{
	background-color: rgb(219, 210, 200);
}

QPushButton:pressed
{
	background-color: rgb(187, 173, 160);
	padding-left: 3px;
	padding-top: 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>ldtID</tabstop>
  <tabstop>ldtPassword</tabstop>
  <tabstop>pBtnLogin</tabstop>
  <tabstop>pBtnLogon</tabstop>
  <tabstop>pBtnForget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ldtID</sender>
   <signal>returnPressed()</signal>
   <receiver>ldtPassword</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>684</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>684</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
